pkgs/development/python-modules/frida-python/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchurl, fetchPypi, buildPythonPackage, typing-extensions, }:
+let
+  version = "16.0.19";
+  devkit = fetchurl {
+    url =
+      "https://github.com/frida/frida/releases/download/${version}/frida-core-devkit-${version}-linux-${stdenv.hostPlatform.linuxArch}.tar.xz";
```

```suggestion
    url = "https://github.com/frida/frida/releases/download/${version}/frida-core-devkit-${version}-linux-${stdenv.hostPlatform.linuxArch}.tar.xz";
```
please hardcode linuxArch since otherwise we get a hash mismatch

---

Done.